ansible/module_utils/port_utils.py

```diff
@@ -316,7 +316,7 @@ def get_port_alias_to_name_map(hwsku, asic_name=None):
         elif hwsku == "Cisco-8102-C64":
             for i in range(0, 64):
                 port_alias_to_name_map["etp%d" % i] = "Ethernet%d" % (i * 4)
-        elif hwsku in ["Cisco-8101-T32", "Cisco-8101-O32", "Cisco-8111-C32", "Cisco-8111-O32"]:
+        elif hwsku in ["Cisco-8101-T32", "Cisco-8101-O32", "Cisco-8111-C32", "Cisco-8111-O32", "32x400Gb"]:
```

Comment:
Will we real have a new SKU naming 32x400Gb? It seems not formal.

Reply:
@wsycqyz , This is the default HwSKU of 8101. This is used internally in cisco runs.

Reply:
@rraghav-cisco then I think it is not suit for upstreaming those code. suggest just used in Cisco's internal branch.